cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_controller.go

```diff
@@ -24,13 +24,13 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/client-go/tools/cache"
-	"sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+	"sigs.k8s.io/cluster-api/pkg/apis/machine/v1beta1"
```

Comment:
this should be github.com/openshift/cluster-api

Reply:
Fixed in https://github.com/openshift/kubernetes-autoscaler/pull/29/commits/2c6b5ed8cbb5665ba1939a0c6ff3c6e02e76d18a and https://github.com/openshift/kubernetes-autoscaler/pull/29/commits/e53ed5424aa58b3ef859da17059ac35404a1b3c6